projects/1.20/assets/silent-gear/silentgear/lang/zh_cn.json

```
@@ -0,0 +1,950 @@
+    "advancements.silentgear.blueprint_pickaxe.description": "制作一个镐的蓝图或模具",
+    "advancements.silentgear.blueprint_pickaxe.title": "蓝图！",
+    "advancements.silentgear.blueprint_sword.description": "制作一个剑的蓝图或模具",
+    "advancements.silentgear.blueprint_sword.title": "剑！",
+    "advancements.silentgear.bow.description": "制作一把寂静装备的弓",
+    "advancements.silentgear.bow.title": "更好的弓",
+    "advancements.silentgear.crafting_station.description": "制作一个具有更多趁手特性的合成站！",
+    "advancements.silentgear.crafting_station.title": "升级我的工作台！",
+    "advancements.silentgear.crimson_iron.description": "在下界采掘一些绯红铁矿石，然后烧炼它们！",
+    "advancements.silentgear.crimson_iron.title": "绯红铁",
+    "advancements.silentgear.crimson_repair_kit.description": "制作一个绯红修复套件",
+    "advancements.silentgear.crimson_repair_kit.title": "精打细算",
+    "advancements.silentgear.crimson_steel.description": "制作一块绯红钢锭！",
+    "advancements.silentgear.crimson_steel.title": "还嫌不够强吗？",
+    "advancements.silentgear.crossbow.description": "制作一架寂静装备的弩",
+    "advancements.silentgear.crossbow.title": "想要来个弩吗？",
+    "advancements.silentgear.crude_tool.description": "用坚韧木杆打造一件工具。在你手头没有蓝图或模具时，它能派上大用场。",
+    "advancements.silentgear.crude_tool.title": "看着粗又糙，效率贼拉高",
+    "advancements.silentgear.grader_catalyst_2.description": "获取一个2级催化剂（灼焰粉）",
+    "advancements.silentgear.grader_catalyst_2.title": "2级分级催化剂",
+    "advancements.silentgear.grader_catalyst_3.description": "获取一个3级催化剂（灿熠粉）",
+    "advancements.silentgear.grader_catalyst_3.title": "3级分级催化剂",
+    "advancements.silentgear.high_durability.description": "打造一件耐久度至少为16,000的工具。",
+    "advancements.silentgear.high_durability.title": "近乎不毁！",
+    "advancements.silentgear.iron_rods.description": "制作一些铁杆。要知道木棍并不是唯一一种能用来做把手的东西……",
+    "advancements.silentgear.iron_rods.title": "耐用的工具手柄",
+    "advancements.silentgear.kachink1.description": "损耗一件装备，使其达到一个特定数值。在该数值下，装备的参数会被重新计算",
+    "advancements.silentgear.kachink1.title": "咔嚓！第一章",
+    "advancements.silentgear.kachink2.description": "在一件拥有【易碎】属性的装备上损耗一点额外耐久度",
+    "advancements.silentgear.kachink2.title": "咔嚓！第二章",
+    "advancements.silentgear.katana.description": "制作一把武士刀",
+    "advancements.silentgear.katana.title": "武士刀",
+    "advancements.silentgear.machete.description": "制作一把砍刀。既是剑，又是斧，也是镰刀！",
+    "advancements.silentgear.machete.title": "砍刀",
+    "advancements.silentgear.material_grader.description": "制作一台材料分级器，激发材料潜能；并获取一个1级催化剂（辉耀粉）",
+    "advancements.silentgear.material_grader.title": "材料分级器",
+    "advancements.silentgear.mattock.description": "制作一把鹤嘴锄，满足你所有的耕作需求！",
+    "advancements.silentgear.mattock.title": "锄头丧气",
+    "advancements.silentgear.mixed_materials.description": "打造一件主要部件类型大于一种的装备",
+    "advancements.silentgear.mixed_materials.title": "混合材料",
+    "advancements.silentgear.moonwalker.description": "装备一双具有【月行者】属性的靴子然后猛地一跳！",
+    "advancements.silentgear.moonwalker.title": "牛顿是哪位？",
+    "advancements.silentgear.nether.description": "我往深处去……",
```

怎么感觉像某首Yogscast的mc金曲
见https://www.bilibili.com/video/BV1sx411T7Ua 的4:40

```suggestion
    "advancements.silentgear.nether.description": "我要搬到下界去……",
```